packages/tizen_audio_manager/tizen/src/tizen_audio_manager_plugin.cc

```diff
@@ -273,41 +181,21 @@ class AudioManagerTizenPlugin : public flutter::Plugin {
             [this](const flutter::EncodableValue *arguments,
                    std::unique_ptr<flutter::EventSink<>> &&events)
                 -> std::unique_ptr<flutter::StreamHandlerError<>> {
-              LOG_INFO("OnListen");
               this->RegisterObserver(std::move(events));
               return nullptr;
             },
             [this](const flutter::EncodableValue *arguments)
                 -> std::unique_ptr<flutter::StreamHandlerError<>> {
-              LOG_INFO("OnCancel");
               this->UnregisterObserver();
               return nullptr;
             });
 
     event_channel->SetStreamHandler(std::move(event_channel_handler));
   }
 
-  static void VolumeChangedCb(sound_type_e type, unsigned int volume,
-                              void *user_data) {
-    auto plugin = static_cast<AudioManagerTizenPlugin *>(user_data);
-    std::string type_str;
-
-    if (ConvertSoundTypeToString(type, type_str)) {
-      flutter::EncodableMap msg;
-      msg[flutter::EncodableValue("type")] = flutter::EncodableValue(type_str);
-      msg[flutter::EncodableValue("level")] =
-          flutter::EncodableValue(static_cast<int64_t>(volume));
-      plugin->events_->Success(flutter::EncodableValue(msg));
-    } else {
-      LOG_ERROR("Failed to convert sound type to string.");
-      plugin->events_->Error("UnknownSoundType",
-                             "Failed to convert sound type to string");
-    }
-  }
-
-  std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> events_;
-  bool volume_changed_cb_is_registered_ = false;
-  int volume_changed_cb_id_ = 0;
+  std::unique_ptr<FlMethodResult> result_;
+  std::unique_ptr<FlEventSink> events_;
+  AudioManager audio_manager_;
 };
 
 void AudioManagerTizenPluginRegisterWithRegistrar(
```

Comment:
nit: Maybe `AudioManagerTizen` → `TizenAudioManager`

(`pubspec.yaml` should be updated as well if you rename the function and the class.)